content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -0,0 +1,142 @@
+---
+id: enrich-your-reference-entity-records
+themes: products-super-power, manage-your-reference-entities
+title: "Enrich your reference entity records"
+popular: false
+ee-only: right
+related:
+---
+
+# Enrich your reference entity records
+
+## Add a record
+Once you’ve created a reference entity, you can add **records**.
+Let’s say that you have just created the reference entity `Brand`, you have defined its records attributes and now it is time to add new records. For example: Fermob, Kartell, Muuto, Fatboy…
+
+To add a new brand to the reference entity `Brand`:
+1. Go to your `Brand` entity (you are automatically on the records grid)
+1. Then click on the `Add new` button, choose a label for the record (the code is automatically generated but you can change it if needed).
+
+![Create a record](../img/Records_creation.png)
+
+::: tips
```

Comment:
This tips seems long to me, you could have simply said:
:::tips
If you have several records to add at once, tick the `Create another record` button at the bottom of the form. Once your first record is added, the PIM will display you the same form to add other records.
When adding the last record, untick the `Create another record` box and save your last record.
:::

Reply:
We want to highlight these steps, because it's the first time we propose this kind of navigation in the PIM. So I prefere keeping it like that ;)